src/components/sections/hero/hero.module.css

```diff
@@ -1,84 +1,39 @@
 .wrapper {
   display: flex;
   flex-direction: column;
-  align-items: center;
   overflow: hidden;
-}
-
-.secondaryColor {
-  padding: 7.5rem 2rem;
+  padding: var(--padding-l) 0rem;
 
-  @media (min-width: 640px) {
-    text-align: center;
-    padding: 7.5rem 3rem;
+  @media (max-width: 834px) {
+    padding: var(--padding-l) 0rem;
   }
 
-  @media (min-width: 1024px) {
-    padding: 15rem 7.5rem;
-  }
-}
-
-.primaryColor {
-  background-color: var(--background-bg-light-primary);
-  padding: 7.5rem 2rem 5rem 2rem;
-
-  @media (min-width: 640px) {
-    padding: 7.5rem 3rem 5rem 3rem;
-  }
-
-  @media (min-width: 1024px) {
-    padding: 7.5rem 2rem 5rem 2rem;
+  @media (max-width: 425px) {
+    padding: 0rem;
   }
 }
 
 .shared {
   width: 100%;
-  max-width: 1200px;
+  max-width: 70.125rem;
```

Comment:
We do have a max-content-width variable in global.css, should we perhaps use that one? Or is the width of the hero-section different from that?

Reply:
It was a bit different